data-prepper-plugins/obfuscate-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/obfuscation/CommonPattern.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.processor.obfuscation;
+
+enum CommonPattern {
+    EMAIL_ADDRESS("[A-Za-z0-9+_.-]+@([\\w-]+\\.)+[\\w-]{2,4}"),
+    IP_ADDRESS("((25[0-5]|(2[0-4]|1\\d|[1-9]|)\\d)\\.?\\b){4}"),
```

Comment:
Maybe name this `IP_ADDRESS_V4`?

Reply:
This pattern is only for IPv4, I was thinking that will have another one for IP_ADDRESS_V6 while this one is still called IP_ADDRESS as we normally mean IPv4 when talking about IP address. 

But it's confusing. So happy to change the name to `IP_ADDRESS_V4` in the next commit.